### PR TITLE
Added client bandwidth measurement result callback.

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: include/freerdp/autodetect.h [hunk 1]
```diff
@@ -52,7 +53,8 @@ struct rdp_autodetect
 	ALIGN64 pBandwidthMeasureStop BandwidthMeasureStop; /* 19 */
 	ALIGN64 pBandwidthMeasureResults BandwidthMeasureResults; /* 20 */
 	ALIGN64 pNetworkCharacteristicsResult NetworkCharacteristicsResult; /* 21 */
-	UINT64 paddingB[32 - 22]; /* 22 */
+	ALIGN64 pClientBandwidthMeasureResult ClientBandwidthMeasureResult; /* 22 */
+	UINT64 paddingB[32 - 21]; /* 23 */
```
Comment (mfleisz) [inline include/freerdp/autodetect.h]: Shouldn't that be UINT64 paddingB[32 - 23]; /* 23 */

Comment (bmiklautz) [inline include/freerdp/autodetect.h]: Same remark from me ;)

**Comment by bmiklautz** [comment]
@freerdp-bot test.

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
+1

**Comment by hardening** [comment]
+1

**Pull request merged**